region/brinstar/green/Noob Bridge aka A Bridge Too Far.json

```diff
@@ -195,11 +120,41 @@
     {
       "from": 2,
       "to": [
-        {"id": 1}
+        {"id": 1},
+        {"id": 2}
       ]
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway",
+      "notable": false,
+      "requires": [],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 1,
+          "openEnd": 1
+        }
+      }
+    },
+    {
+      "link": [1, 1],
+      "name": "Leave Shinecharged",
+      "notable": false,
+      "requires": [
+        {"canShineCharge": {
+          "usedTiles": 33,
+          "openEnd": 2
+        }},
+        "canShinechargeMovement"
+      ],
+      "exitCondition": {
+        "leaveShinecharged": {
+          "framesRemaining": 120
```

Comment:
At full run I was seeing ~140, but when shortcharging I could get 150.
So there is a full run vs shortcharge difference, and that would be something to cover with a `h_shinechargeLeniency` in the future.

Reply:
There are a lot of other factors too that could justify shinecharge leniency:
- tendency for less experienced players to idle for a moment after getting the shinecharge, before beginning moving
- lack of knowledge/ability of how to ledge-grab, down-back, or other movement techniques
- lack of knowledge/ability to quickly hone in accurately on the place where the spark should be activated, e.g. by stopping on a dime or carefully controlling a jump.

I think we could treat this kind of like heat frames, where we put tight values into the strats, and based on settings, the randomizer assumes you need extra frames available for leniency, e.g. maybe something like this:

Basic: 60 frames
Medium: 30 frames
Hard: 20 frames
Very Hard: 10 frames
Expert: 5 frames
Extreme: 2 frames
Insane: 0 frames

In this case, the leniency wouldn't need to be explicitly written into the strats.